astropy/units/function/logarithmic.py

```diff
@@ -216,12 +216,12 @@ class LogQuantity(FunctionQuantity):
     subclasses, as in::
 
         >>> import astropy.units as u
-        >>> u.Magnitude(15.)
-        <Magnitude 15.0 mag>
-        >>> u.Magnitude(10.*u.count/u.second)
+        >>> u.Magnitude(12.5)  # doctest: +FLOAT_CMP
```

Comment:
I think this particular one is actually not necessary -- the real reason is to ensure that `nnn.0` and `nnn.` are treated as being the same -- and here I changed to `nnn.5` for that reason.